cmd/node/config/config.toml

```diff
@@ -625,7 +625,8 @@
             MessagesMarshalizer = "json"
             MaxLoopTime = 1000
     [VirtualMachine.Querying]
-        OutOfProcessEnabled = false
+        OutOfProcessEnabled = true
```

Comment:
we could simply delete from here OutOfProcess and have it default on true directly when creating the scquery service

Reply:
good idea, will hardcoded it to true